en/docs/modules/misc/VideoChip.md

```diff
@@ -33,8 +33,14 @@ end
 ### SetPixel(position `vec2`, color `color`)
 Sets the pixel at the specified position to the specified **color**.
 
+### SetPixelData(PixelData `PixelData`)
+⚠️ **This feature is new, as of Retro Gadgets 0.1.5, and not fully tested yet.**  
```

Comment:
Slight grammar change
```suggestion
⚠️ **This feature is new as of Retro Gadgets 0.1.5, and not fully tested yet.**  
```
We really should be testing things we add, but oh well.

Reply:
I did test all of the things I wrote, I just added that because it's new and undocumented so things might change in the future. Maybe it should be changed to something along the lines of: `This feature is new and undocumented as of 0.1.5, things might change in the future`